internal/backend/db/db.go

```diff
@@ -13,20 +12,6 @@ import (
 	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
 )
 
-type Transaction interface {
```

Comment:
Maybe it's worth looking at https://github.com/Thiht/transactor

Reply:
maybe in the future...